requirements.txt

```diff
@@ -4,4 +4,5 @@ django-environ==0.8.1
 idna==3.3
 requests==2.28.1
 requests-toolbelt==0.9.1
+types-requests==2.28.9
```

Comment:
We probably don't need this line now we've got it in the pre-commit (pre-commit handles their own dependency/install jazz); note the lack of mypy in this file